README.md

```diff
@@ -65,6 +65,23 @@ class MyEventEmitter<T> extends (EventEmitter as { new<T>(): TypedEmitter<T> })<
 }
 ```
 
+## RxJS `fromEvent` types inference
+
+The default `fromEvent` from RxJS will return an `Observable<unknown>` for our typed emitter.
+
+This can be fixed by the following code, by replacing the `fromEvent` type with our enhanced one: `FromEvent`:
+
+```ts
+import { fromEvent as rxjsFromEvent } from "rxjs"
```

Comment:
Really tiny suggestion: `rxjsFromEvent` is quite a handful. What would you think about renaming the import alias to `rxFromEvent`?

It's just two letters difference, but I think it would make the code feel "friendlier".

Reply:
Absolutely agree with this.

Have renamed `rxjsFromEvent` to `rxFromEvent` from both the README and the source code.